lottie/src/main/java/com/airbnb/lottie/model/content/LBlendMode.java

```diff
@@ -0,0 +1,92 @@
+package com.airbnb.lottie.model.content;
+
+import android.os.Build;
+import androidx.annotation.Nullable;
+import androidx.annotation.RequiresApi;
+import androidx.core.graphics.BlendModeCompat;
+
+/**
+ * Lottie BlendMode,
+ * not to be confused with Paint.BlendMode in android graphics core,
+ * which we will rely on for rendering.
+ */
+public enum LBlendMode {
+  NORMAL,
+  MULTIPLY,
+  SCREEN,
+  OVERLAY,
+  DARKEN,
+  LIGHTEN,
+  COLOR_DODGE,
+  COLOR_BURN,
+  HARD_LIGHT,
+  SOFT_LIGHT,
+  DIFFERENCE,
+  EXCLUSION,
+  HUE,
+  SATURATION,
+  COLOR,
+  LUMINOSITY,
+  ADD,
+  HARD_MIX;
+
+  @Nullable
+  public BlendModeCompat toNativeBlendMode() {
+    switch (this) {
+      case NORMAL:
+        return null; // BlendMode.CLEAR?
+      case SCREEN:
+        return BlendModeCompat.SCREEN;
+      case OVERLAY:
+        return BlendModeCompat.OVERLAY;
+      case DARKEN:
+        return BlendModeCompat.DARKEN;
+      case LIGHTEN:
+        return BlendModeCompat.LIGHTEN;
+      case ADD:
+        return BlendModeCompat.PLUS;
+      default:
```

Comment:
Instead of having `toNativeBlendModeQ` and landing it without any usages and this code commented, I would do the following:
```
switch (type) {
	case NORMAL:
		return null;
	case SCREEN:
		return BlendModeCompat.SCREEN;
	…
	// Blend modes below were not added to the platform until Q.
	// To prevent unexpected issues where animations look correct
	// during development but silently break for users with older devices
	// we won't support any of these until Q is widely used.
	case MULTIPLY:
	case COLOR_DODGE
	…
		return null;
```

If there was some complexity in the mapping, I could see the value in landing the code you write but because the mapping is trivial, I would prefer to not land unused code like that.